src/picongpu/include/simulation_defines/param/speciesDefinition.param

```diff
@@ -85,6 +112,9 @@ typedef bmpl::vector<
     current<UsedParticleCurrentSolver>,
     massRatio<MassRatioElectrons>,
     chargeRatio<ChargeRatioElectrons>
+#if(ENABLE_SYNCHROTRON_PHOTONS == 1)
```

Comment:
*Offline* discussion with @psychocoderHPC:
However this pre-compiler flag is used as switch here. 
A user can add the attribute on its own to add photon-emitting-electrons. 
A pre-compiler flag is only used for cmake compile tests, not as switch from your param file.